packages/app/cypress/e2e/cypress-in-cypress.cy.ts

```diff
@@ -84,5 +92,72 @@ describe('Cypress in Cypress', { viewportWidth: 1500, defaultCommandTimeout: 100
         expect(ctx.actions.project.launchProject).to.have.been.called
       })
     })
+
+    it(`scales the AUT correctly in ${testingType}`, () => {
+      cy.scaffoldProject('cypress-in-cypress')
+      cy.findBrowsers()
+      cy.openProject('cypress-in-cypress')
+      cy.withCtx((ctx) => {
+        ctx.coreData.localSettings.preferences.reporterWidth = 800
+      })
+
+      cy.startAppServer(testingType)
+      cy.visitApp()
+
+      cy.get('[data-cy="spec-item"]').first().click()
+      // Let runner stabilize
+      cy.get('#unified-reporter').should('be.visible')
+
+      // validate that the width we set in `withCtx` above is the starting point
+      cy.get(`[data-cy="reporter-panel"]`).invoke('outerWidth').should('eq', 800)
+      cy.percySnapshot('initial state')
+
+      // we will move the right-hand handle of the Reporter
+      // to these positions from the left of the screen
+      const dragPositions = [1000, 1090, 900, 600]
+
+      // based on viewport sizes for CT and e2e tests in the `cypress-in-cypress`
+      // projects, we expect certain scale % values to be shown
+      const testingTypeExpectedScales = {
+        component: ['93%', '75%'],
+        e2e: ['46%', '37%', '56%', '85%'],
+        componentShortViewport: '61%',
+        e2eShortViewport: '46%',
+      }
+
+      // resize the reporter using each of the dragPositions and take Percy snapshots
+      dragPositions.forEach((position, index) => {
+        dragHandleToClientX('panel2', position).then(() => {
+          const expectedScale = testingTypeExpectedScales[testingType][index]
+
+          if (expectedScale) {
+            cy.contains(expectedScale).should('be.visible')
+          } else {
+            cy.contains('%)').should('not.exist')
```

Comment:
Due to the different default viewport sizes of CT and E2E, as we make the reporter smaller, we hit the 100% state earlier in CT. `testingTypeExpectedScales.component` has a length of 2.

I wondered about whether this approach is truly ideal, vs having this in two separate specs, because of the different outcomes. For now, I'll add a couple of comments to keep things clear... it's a lot of comments, but it's pretty hard to work backwards from the code and understand why the feature works this way.

Reply:
Ok, seems fine for now :+1: